playbooks/capsule-dev/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
just the file paths and the permissions change, here is the diff

``` 
19c20
<   shell: grep oauth_consumer_key /home/vagrant/foreman/config/settings.yaml | cut -d ' ' -f 2
---
>   shell: grep oauth_consumer_key /etc/foreman/settings.yaml | cut -d ' ' -f 2
24c25
<   shell: grep oauth_consumer_secret /home/vagrant/foreman/config/settings.yaml | cut -d ' ' -f 2
---
>   shell: grep oauth_consumer_secret /etc/foreman/settings.yaml | cut -d ' ' -f 2
41,44d41
< 
< - name: 'Change cert permissions'
<   shell: chmod 755 -R /etc/pki/katello/private
<   delegate_to: "{{ server }}"
```
maybe we could make a file base path variable and use that?